WalletWasabi/WalletWasabi.csproj

```diff
@@ -20,8 +20,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="NBitcoin" Version="4.1.1.98" />
-  </ItemGroup>
+	<PackageReference Include="NBitcoin" Version="4.1.1.98" />
+    <PackageReference Include="Microsoft.Win32.Registry" Version="4.5.0" />  </ItemGroup>
```

Comment:
Could you remove this package and use copypaste programming the relevant parts of the code?

Reply:
This one? https://github.com/dotnet/corefx/tree/master/src/Microsoft.Win32.Registry
It is a dotnet core library still copypaste it?

Reply:
Yes.

Reply:
I give up. Started to go down on the rabbit hole but it is just getting deeper and deeper. For example Advapi32 class contains more than 100 files and multiplatform adds more complexity.
![image](https://user-images.githubusercontent.com/9844978/56272409-efa26a80-60fa-11e9-8372-cef8b6c81122.png)

Reply:
And also one more problem if you add everything to the project the compiler will give you a lot of "already defined" errors because they are part of the dotnet core library so you must precisely truncate the corresponding code or you will get a lot error.